### PR TITLE
convert SID to sid - get rid of external tr call

### PR DESCRIPTION
**Pull request by PeterPitterling**
add sid to GLOBALS - was anyway used like a GLOBAL variable

### CODE REVIEW

File: ra/SAPHana [hunk 1]
```diff
@@ -798,7 +798,7 @@ function saphana_init() {
     export SAPSYSTEMNAME=$SID
     HANA_CALL_TIMEOUT="${OCF_RESKEY_HANA_CALL_TIMEOUT:-60}"
     super_ocf_log debug "DBG: Used new method to get SID ($SID) and InstanceNr ($InstanceNr)"
-    sid=$(echo "$SID" | tr [:upper:] [:lower:])
+    sid="$SID"
```
Comment (fmherschel) [inline ra/SAPHana:801]: I prefer to do that more explicitly like: sid="${SID,,}"

File: ra/SAPHana [hunk 2]
```diff
@@ -2863,8 +2863,9 @@ function saphana_demote_clone() {
 #
 
 ## GLOBALS
-SID=""
-sidadm=""
+SID=''
+declare -l sid=''
```
Comment (fmherschel) [inline ra/SAPHana:2867]: Same as above.

File: ra/SAPHanaTopology [hunk 3]
```diff
@@ -492,7 +492,7 @@ function sht_init() {
     InstanceName="HDB${InstanceNr}"
     HANA_CALL_TIMEOUT="${OCF_RESKEY_HANA_CALL_TIMEOUT:-60}"
     super_ocf_log debug "DBG2: Used new method to get SID ($SID) and InstanceNr ($InstanceNr)"
-    sid=$(echo "$SID" | tr [:upper:] [:lower:])
+    sid="$SID"
```
Comment (fmherschel) [inline ra/SAPHanaTopology:495]: Same as for SAPHana (do that explicitly) : sid="${SID,,}"

File: ra/SAPHanaTopology [hunk 4]
```diff
@@ -1165,8 +1165,9 @@ function sht_notify() {
 #
 
 ## GLOBALS
-SID=""
-sidadm=""
+SID=''
+declare -l sid=''
```
Comment (fmherschel) [inline ra/SAPHanaTopology:1169]: Same as above.

**Review by fmherschel — CHANGES_REQUESTED** [review]
would like to be more explicit (see remarks)

File: ra/SAPHana (hunk 1 above)
Comment (Pitterling) [reply ra/SAPHana:801]: SID,, and SID^^ are only available since BASHv4 .. SLES11 and RHEL6 are shipped with BASHv3 ..

**Comment by fmherschel** [comment]
BTW: declare -l <var> is listed as new bash4.x feature at: https://wiki.bash-hackers.org/bash4 (not sure, if that is correct)

**Comment by PeterPitterling** [comment]
true, just checked myself .. thought i did that already ... declare -l is only available with bash4

just verified RHEL6.10 has bash v4.1 .. but SLES11.4 latest is 3.2.57

In case SLES11 is out of scope go ahead with more intuitive sid="${SID,,}"

**Review by fmherschel — COMMENTED** [review]
Looks good for me, lets also check abriel.

**Review by angelabriel — APPROVED** [review]
Looks good for me, thanks

**Review by angelabriel — APPROVED** [review]
Looks good, thanks

**Review by fmherschel — APPROVED** [review]
Looks good for me now :)

**Pull request merged**